test_cursor_pr_review.py

```
@@ -154,6 +156,22 @@ def test_create_github_workflow_anthropic(self):
         env = workflow["jobs"]["ai-review"]["steps"][-1]["env"]
         assert "ANTHROPIC_API_KEY" in env
 
+class TestCodeRabbitConfig:
+    """Test CodeRabbit configuration generation."""
+    
+    def test_create_coderabbit_config(self):
+        """Test CodeRabbit config generation."""
+        config = ReviewConfig(
```

Ruff (0.11.9)

Local variable `config` is assigned to but never used

Remove assignment to unused variable `config`

(F841)